### ISSUE TITLE
Package control not installing packages

### ISSUE THREAD

**Issue by jmona789** [wbond/package_control#1082]
When I try to use package control to install a package nothing happens. This is what my sublime console is showing:

```
startup, version: 3103 osx x64 channel: stable
executable: /Applications/Sublime Text.app/Contents/MacOS/Sublime Text
working dir: /
packages path: /Users/JamesMona/Library/Application Support/Sublime Text 3/Packages
state path: /Users/JamesMona/Library/Application Support/Sublime Text 3/Local
zip path: /Applications/Sublime Text.app/Contents/MacOS/Packages
zip path: /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages
ignored_packages: []
pre session restore time: 0.181345
using gpu buffer for window
first paint time: 0.246111
using gpu buffer for window
using gpu buffer for window
using gpu buffer for window
using gpu buffer for window
using gpu buffer for window
startup time: 0.538784
error loading /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/0_package_control_loader.sublime-package: File is not a zip file
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.profile_syntax_definition
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Python.syntax_test_python
reloading plugin Vintage.vintage
reloading plugin Vintage.vintage_commands
reloading plugin Vintage.vintage_motions
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 162, in on_api_ready
    m.plugin_loaded()
  File "2_bootstrap in /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 35, in plugin_loaded
  File "package_control.loader in /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 113, in exists
  File "./zipfile.py", line 933, in __init__
  File "./zipfile.py", line 974, in _RealGetContents
zipfile.BadZipFile: File is not a zip file
Package Control: Not running package cleanup since bootstrapping is not yet complete
Package Control: Fetching list of available packages
  Platform: osx-x64
  Sublime Text Version: 3103
  Package Control Version: 3.1.2
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.33.29
  Resolved IPv6: 2600:3c02::f03c:91ff:fedf:6883
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found previously exported CA bundle at /Users/JamesMona/Library/Application Support/Sublime Text 3/Packages/User/Package Control.system-ca-bundle
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /Users/JamesMona/Library/Application Support/Sublime Text 3/Packages/User/Package Control.merged-ca-bundle
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher DHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=codexns.io,O=Codex Non Sufficit LC,L=Newbury,S=Massachusetts,C=US,serialNumber=471714639,businessCategory=Private Organization,incorporationState=Massachusetts,incorporationCountry=US
    subject alt name: codexns.com, packagecontrol.io, wbond.net, rc.codexns.io, codexns.io, www.codexns.io
    expire date: Nov 27 23:59:59 2016 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  User-Agent: Package Control v3.1.2
  Accept-Encoding: gzip,deflate
  If-Modified-Since: Sat, 13 Feb 2016 21:00:07 GMT
  Host: packagecontrol.io
  Connection: Keep-Alive
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 304 Not Modified
  Server: nginx
  Date: Sat, 13 Feb 2016 21:09:07 GMT
  Last-Modified: Sat, 13 Feb 2016 21:00:07 GMT
  Connection: keep-alive
  ETag: "56bf9957-24e94d"
  Strict-Transport-Security: max-age=15552000
Package Control: Using cached content for https://packagecontrol.io/channel_v3.json from /Users/JamesMona/Library/Application Support/Sublime Text 3/Packages/User/Package Control.cache/01524fae79697630d0454ba3fabd9414
Traceback (most recent call last):
  File "package_control.package_installer in /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 159, in on_done
  File "package_control.package_disabler in /Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/Package Control.sublime-package", line 74, in disable_packages
ImportError: No module named 'package_control'
Package Control: Urllib HTTPS Debug General
  Closing connection to packagecontrol.io on port 443 after 1 request
```

**Comment by kostrovart** [comment]
I have the same issue on Linux :-(
ImportError: No module named 'package_control'

**Comment by wbond** [comment]
@jmona789 It appears that `/Users/JamesMona/Library/Application Support/Sublime Text 3/Installed Packages/0_package_control_loader.sublime-package` is somehow invalid. Thus it need to be deleted so that Package Control can regenerate it properly.

**Comment by wbond** [comment]
This should be resolved by 0cd0b47f5c2254bf969081edf52bdaba834d04b6
